src/main/java/org/opentestsystem/ap/imrt/common/model/ImrtItem.java

```diff
@@ -2,26 +2,30 @@
 
 import javax.persistence.DiscriminatorValue;
 import javax.persistence.Entity;
-import javax.persistence.Transient;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
 
 /**
  * An IMRT Item containing the IMRT hierarchy
  */
 @Entity
 @DiscriminatorValue("item")
 public class ImrtItem extends BaseItem {
-    @Transient
-    private StimulusLink stimulusLink;
+    @JoinTable(
+            name = "stim_link",
+            joinColumns = @JoinColumn(name = "item_key"),
+            inverseJoinColumns = @JoinColumn(name = "item_key_stim")
+    )
+    @ManyToOne
+    private Stimulus stimulus;
```

Comment:
`stim_link` is a join table.  The earlier issues were caused because of the audit fields.  After some discussion with @jeffjohnson9046 we both agreed this table didn't need the audit columns since they can be determined by the joined entities.